app/Commands/Servers/RebootCommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Commands\Servers;
+
+use App\Commands\BaseCommand;
+
+class RebootCommand extends BaseCommand
+{
+    protected $signature = 'servers:reboot
+                            {server_id? : The server to reboot}
+                            {--all : Reboot all servers}
+                            {--force : Force reboot}
+                            {--profile=}';
+
+    protected $description = 'Reboot a server';
+
+    public function action(): int
+    {
+        if ((bool) $this->option('all')) {
+            $this->rebootAllServers();
+            return self::SUCCESS;
+        }
+
+        $serverId = $this->argument('server_id');
+
+        if (empty($serverId)) {
+            $serverId = $this->askToSelectServer('Which server would you like to reboot?');
+        }
+
+        $server = $this->spinupwp->servers->get($serverId);
+
+        $reboot = (bool) $this->option('force') || $this->confirm("Are you sure you want to reboot \"{$server->name}\"?", 'yes');
+
+        if (!$reboot) {
+            return self::SUCCESS;
+        }
+
+        $this->rebootServers([$server]);
+
+        return self::SUCCESS;
+    }
+
+    protected function rebootAllServers(): void
+    {
+        $reboot = (bool) $this->option('force') || $this->confirm('Are you sure you want to reboot all servers?', 'yes');
+        if ($reboot) {
+            $this->rebootServers($this->spinupwp->servers->list());
+        }
+    }
+
+    protected function rebootServers($servers): void
+    {
+        if (empty($servers)) {
+            return;
+        }
+
+        foreach ($servers as $server) {
+            $response = $server->reboot();
+            $this->info("Rebooting server {$server->name}. Event ID: {$response}");
```

Comment:
This needs to handle servers that are already being rebooted:

<img width="702" alt="CleanShot 2022-01-11 at 20 21 46@2x" src="https://user-images.githubusercontent.com/1422996/149087957-9b6ce1ca-b349-4743-acb7-bef1036ba14b.png">

Reply:
Can we copy the success output I introduced in: https://github.com/deliciousbrains/spinupwp-cli/blob/main/app/Commands/Servers/DeleteCommand.php#L36

- [ ] This should list all servers that were successfully queued.
- [ ] Servers that threw an error should not be listed.
- [ ] If no servers were successful we should show an error.